.github/workflows/test.yml

```diff
@@ -31,3 +31,15 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: make test-all
+
+      ###########################################################
+      # Notifying #contributors about test failure on main branch
+      ###########################################################
+      - name: failure run notification
+        if: ${{ github.ref == 'refs/heads/main' && failure() }}
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_LAUNCHING_CHANNEL_WEBHOOK }}
+        uses: Ilshidur/action-slack@2.1.0
```

Comment:
This seems pretty old.

Can we use something more recently updated like https://github.com/ravsamhq/notify-slack-action ?

Reply:
I am not familiar with that action, maybe you can take over this PR?

Reply:
This is the one that I am using in my org, but it works pretty well so far. I dont know why we care about the `old` part.

Reply:
Mostly security reasons.

See their repo has open prs from dependabot but no one has merged.

https://github.com/Ilshidur/action-slack/pulls

If we're going to add more automation, i think it would be wise to limit blast radius as much as possible. The action im proposing is very similar to the one you're proposing. The only difference is that it is more up to date and therefore more likely to be up to date in the future.

Reply:
As another solution, slack official github action seems better.

https://github.com/slackapi/slack-github-action#usage-2

Reply:
Nice find! Up to date and official. This is my favorite one so far. Thank you for sharing